templates/terraform/examples/app_engine_standard_app_version.tf.erb

```diff
@@ -17,6 +17,13 @@ resource "google_app_engine_standard_app_version" "<%= ctx[:primary_resource_id]
     port = "8080"
   }
 
+  automatic_scaling {
```

Comment:
Can you throw as many of these fields as possible in here so they are tested?

Preferably we would make a new handwritten test file similar to this one: https://github.com/GoogleCloudPlatform/magic-modules/blob/master/third_party/terraform/tests/resource_app_engine_application_test.go

Where we could add some of these fields, then perform an update to either change the scaling mode and update some of these values.

Reply:
I've added all the scaling fields to this test.

I also tried to create a handwritten test by copying `resource_app_engine_standard_app_version_test.go`. Unfortunately this is my first ever attempt at writing go and I am unable to run the tests locally. 
```
% make testacc TEST=./google TESTARGS='-run=TestAccAppEngineApplication_basic'

==> Checking source code against gofmt...
==> Checking that code complies with gofmt requirements...
TF_ACC=1 TF_SCHEMA_PANIC_ON_ERROR=1 go test ./google -v -run=TestAccAppEngineApplication_basic -timeout 240m -ldflags="-X=github.com/terraform-providers/terraform-provider-google/version.ProviderVersion=acc"
=== RUN   TestAccAppEngineApplication_basic
=== PAUSE TestAccAppEngineApplication_basic
=== CONT  TestAccAppEngineApplication_basic
    TestAccAppEngineApplication_basic: testing.go:635: Step 0 error: errors during apply:

        Error: error creating project tf-test-5223833485568933614 (tf-test-5223833485568933614): googleapi: Error 400: field [parent] has issue [Parent id must be numeric.], badRequest. If you received a 403 error, make sure you have the `roles/resourcemanager.projectCreator` permission

          on /var/folders/wf/xm7lxbtn1nbbvzfzb3r4v89r0000gn/T/tf-test893474397/main.tf line 2:
          (source code not available)


--- FAIL: TestAccAppEngineApplication_basic (1.07s)
FAIL
FAIL	github.com/terraform-providers/terraform-provider-google/google	1.514s
FAIL
```

I can see that the automated build in this PR has failed, so I must have done something wrong. But I cannot see the automated build results and I can't run the tests locally as they appear to be missing the org-id (although I have set the GOOGLE_ORG env var to our domain name).

Can you see what is failing in the CI build or point me to instructions on how to run these tests locally?

Reply:
Awesome, thanks for all the work!

This guide has info on setting the org_id and billing account as environment variables: https://github.com/terraform-providers/terraform-provider-google/blob/master/.github/CONTRIBUTING.md#tests

Reply:
I had already tried to set the env vars `GOOGLE_ORG` to the domain name of our organisation and `GOOGLE_BILLING_ACCOUNT` to the guid-like ID of our billing account. But that doesn't seem to work with the code from the terraform test: https://github.com/wvanderdeijl/magic-modules/blob/app-engine-standard-scaling/third_party/terraform/tests/resource_app_engine_standard_app_version_test.go#L12-L13

```
% make testacc TEST=./google TESTARGS='-run=TestAccAppEngineStandardAppVersion_update'
==> Checking source code against gofmt...
==> Checking that code complies with gofmt requirements...
TF_ACC=1 TF_SCHEMA_PANIC_ON_ERROR=1 go test ./google -v -run=TestAccAppEngineStandardAppVersion_update -timeout 240m -ldflags="-X=github.com/terraform-providers/terraform-provider-google/version.ProviderVersion=acc"
=== RUN   TestAccAppEngineStandardAppVersion_update
=== PAUSE TestAccAppEngineStandardAppVersion_update
=== CONT  TestAccAppEngineStandardAppVersion_update
    TestAccAppEngineStandardAppVersion_update: testing.go:635: Step 0 error: errors during apply:

        Error: error creating project tf-test-appeng-std8o3aqjdj4r (tf-test-appeng-std8o3aqjdj4r): googleapi: Error 400: field [parent] has issue [Parent id must be numeric.], badRequest. If you received a 403 error, make sure you have the `roles/resourcemanager.projectCreator` permission

          on /var/folders/wf/xm7lxbtn1nbbvzfzb3r4v89r0000gn/T/tf-test299260747/main.tf line 2:
          (source code not available)


--- FAIL: TestAccAppEngineStandardAppVersion_update (2.35s)
FAIL
FAIL	github.com/terraform-providers/terraform-provider-google/google	3.443s
FAIL
make: *** [testacc] Error 1
```

I also tried setting GOOGLE_ORG to the numeric id of our organisation, but that throws the same error.

Anyhow; it does work on your CI servers so it must be something I'm doing wrong (or missing from CONTRIBUTING.md) but it doesn't seem relevant to this PR.